### PR TITLE
[Wallet] Add link to additional funding options

### LINKED ISSUE #5021: Add copy to the bottom of add/withdraw screen indicating that there are additional withdraw options
### Expected Behavior

Add footnote to Add/Withdraw screen with the following info:
EN: “For more ways to add and withdraw funds, visit [How to fund your account]”
ES: "Para obtener más formas de agregar y retirar fondos, visite [Cómo depositar fondos en cuenta]"

The link points to valorrapp.com/fund-wallet.

Design: [Figma](https://www.figma.com/file/zt7aTQ5wuXycIwxq5oAmF9/Wallet-Refresh?node-id=3571%3A3793)

Cash In and Out Landing

### Current Behavior

What is the current behavior?

### PR DESCRIPTION
**Pull request by gnardini**
### Description

Added a text on the bottom of the add/withdraw screen indicating that there are additional withdraw options and a link to takes to https://valoraapp.com/fund-wallet

Associated PR on the branding repo: https://github.com/clabs-co/valora-app-branding/pull/4

### Tested

On the simulator.

### Related issues

- Fixes #5021

### CODE REVIEW

File: packages/mobile/branding/celo/src/brandingConfig.ts [hunk 1]
```diff
@@ -3,5 +3,6 @@ export const CELO_FAUCET_LINK = 'https://celo.org/developers/wallet'
 export const CELO_TERMS_LINK = 'https://celo.org/terms'
 export const TOS_LINK = 'https://celo.org/user-agreement'
 export const FAQ_LINK = 'https://celo.org/faq'
+export const FUNDING_LINK = 'https://valoraapp.com/fund-wallet'
```
Comment (gnardini) [inline packages/mobile/branding/celo/src/brandingConfig.ts]: Question: Is it fine to add a link to valoraapp.com here? I think we wanted to keep links to celo.org here, but there is no link to additional funding options there. I added the link in the branding repo as well: https://github.com/clabs-co/valora-app-branding/pull/4

Comment (jeanregisser) [reply packages/mobile/branding/celo/src/brandingConfig.ts]: no big deal, but would prefer to see celo.org here even if it doesn't exist.

```suggestion
// Note: the actual link doesn't exist on celo.org and is overridden by the Valora branding
export const FUNDING_LINK = 'https://celo.org/fund-wallet'
```

**Review by jeanregisser — COMMENTED** [review]
Great!! 👍 

We also need the valora branding commit sha1 update in https://github.com/celo-org/celo-monorepo/blob/cd2dc07462d3f0c520e22f36fff81072bb6bf47f/packages/mobile/scripts/sync_branding.sh#L24

I'll change it.

**Review by jeanregisser — APPROVED** [review]

**Comment by Lss-Ankit** [comment]
@jeanregisser Verified task on latest Android internal build v1.1.0 (1004294311) and Test flight build v1.1.0 (23) and found the following:
Devices: Google Pixle Xl (8.1) , iPhone 11 (14 beta 6),

Tested Scenarios:

1. Verify user is able to see the "How to fund your account" button on "Add and withdraw" page 
2. Verify user should redirect to  "https://valoraapp.com/fund-wallet"

Please let me know if i need to verify anything else.
Thanks!

**Comment by Lss-Ankit** [comment]
@tarikbellamine  Verified task on latest  Test flight build v1.2.0 (24) and found the following:


Tested Scenarios:

- Verify user is able to see the "How to fund your account" button on "Add and withdraw" page
- Verify user should redirect to "https://valoraapp.com/fund-wallet"

Please let me know if i need to verify anything else.
Thanks!

**Comment by Lss-Ankit** [comment]
@tarikbellamine Verified task on latest Android internal build 1.2.0(1004294313) and found the following:

Tested Scenarios:

- Verify user is able to see the "How to fund your account" button on "Add and withdraw" page
- Verify user should redirect to "https://valoraapp.com/fund-wallet"

Please let me know if i need to verify anything else.
Thanks!

**Pull request merged**